app/controllers/users/omniauth_callbacks_controller.rb

```diff
@@ -25,11 +25,19 @@ def google_oauth2
       # update a user's FireCloud status
       @user.delay.update_firecloud_status
       sign_in(@user)
+      @alert = nil
+      # check if user needs to accept updated Terra ToS
+      if @user.must_accept_terra_tos?
+        @alert = 'We have detected that Terra has updated their terms of service - please visit ' \
```

Comment:
Trimmed, and casing [Terms of Service](https://terra.bio/about/terms-of-service/) like Terra does:

```suggestion
        @alert = 'Terra has updated their Terms of Service.  Please visit ' \
```